Runtime/Native/Android/NativeClient.cs

```diff
@@ -431,7 +431,9 @@ public void HandleAnr()
 
                                     NativeReport(AndroidJNI.NewStringUTF(AnrMessage), true);
                                     // update error.type attribute in case when crash happen 
-                                    SetAttribute(ErrorTypeAttribute, CrashType);
+                                    AddAttribute(
```

Comment:
Isn't it more safe to use `SetAttribute` here? We can do the same for `HangType` attribute as well.

It shouldn't increase baseline RAM usage either since we only take this code path when we create an ANR report.

Reply:
The AddAttribute method gonna do some additional checks, that are unnecessary in this situation because we're sure that attribute name and attribute value is always defined.